spec/models/ability_spec.rb

```diff
@@ -10,6 +10,7 @@
     it { is_expected.not_to be_able_to(:create, ContentBlock) }
     it { is_expected.not_to be_able_to(:update, ContentBlock) }
     it { is_expected.to     be_able_to(:read,  ContentBlock) }
+    it { is_expected.to     be_able_to(:view_upload, GenericFile) }
```

Comment:
@terrellt  Why view_upload here?  You are removing the upload button without the create ability and upload ~= create.
Should this be only view?